ql/src/java/org/apache/hadoop/hive/ql/metadata/formatting/MapBuilder.java

```diff
@@ -17,51 +17,53 @@
  */
 package org.apache.hadoop.hive.ql.metadata.formatting;
 
-import java.util.HashMap;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
 /**
  * Helper class to build Maps consumed by the JSON formatter.  Only
  * add non-null entries to the Map.
  */
-public class MapBuilder {
-    private Map<String, Object> map = new LinkedHashMap<String, Object>();
+public final class MapBuilder {
+  private Map<String, Object> map = new LinkedHashMap<String, Object>();
 
-    private MapBuilder() {}
+  private MapBuilder() {
+  }
 
-    public static MapBuilder create() {
-        return new MapBuilder();
-    }
+  public static MapBuilder create() {
+    return new MapBuilder();
+  }
 
-    public MapBuilder put(String name, Object val) {
-        if (val != null)
-            map.put(name, val);
-        return this;
+  public MapBuilder put(String name, Object val) {
+    if (val != null) {
+      map.put(name, val);
     }
+    return this;
+  }
 
-    public MapBuilder put(String name, boolean val) {
-        map.put(name, Boolean.valueOf(val));
-        return this;
-    }
+  public MapBuilder put(String name, boolean val) {
+    map.put(name, Boolean.valueOf(val));
+    return this;
+  }
 
-    public MapBuilder put(String name, int val) {
-        map.put(name, Integer.valueOf(val));
-        return this;
-    }
+  public MapBuilder put(String name, int val) {
+    map.put(name, Integer.valueOf(val));
+    return this;
+  }
 
-    public MapBuilder put(String name, long val) {
-        map.put(name, Long.valueOf(val));
-        return this;
-    }
+  public MapBuilder put(String name, long val) {
+    map.put(name, Long.valueOf(val));
+    return this;
+  }
 
-    public <T> MapBuilder put(String name, T val, boolean use) {
-        if (use)
-            put(name, val);
-        return this;
+  public <T> MapBuilder put(String name, T val, boolean use) {
+    if (use) {
+      put(name, val);
     }
+    return this;
+  }
 
-    public Map<String, Object> build() {
-        return map;
-    }
+  public Map<String, Object> build() {
+    return map;
+  }
```

Comment:
All changes (there are several) in this file are out of scope of the PR.  Please revert.

Reply:
Removed.